### PR TITLE
Features/hdev 265 implement loading bar

### PR DESCRIPTION
**Pull request by psalm-ov**
## Description of Changes

[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDEV-265)

### Developer Checklist:

* [x] I have followed the guidelines in our Contributing document
* [x] This PR has a corresponding JIRA ticket
* [x] My branch conforms with our naming convention i.e. `feature/HDEV-XXX-description`
* [ ] All files have appropriate file headers and documentation
* [ ] I have written new tests for your core changes, as applicable
* [ ] I have successfully ran tests locally

### Reviewers Checklist:
* [ ] Code is readable and understandable; any unclear parts have explanations 
* [ ] UI/UX changes match the corresponding figma/other design resources, if applicable
* [ ] I have successfully ran tests locally

### CODE REVIEW

File: src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx [hunk 1]
```diff
@@ -102,26 +102,40 @@ const BridgeTransactionModal = ({
...
               </div>
             </div>
             <div className="flex justify-between mb-2 font-bold">
               <div className="text-secondary-alternate">Estimated upper bound shuttle fee</div>
               <div>
-                <div>
-                  {upperBoundFee.toFixed(2)} {token.symbol}
+                <div
+                  className={`
+            ${!upperBoundFee && 'animate-pulse bg-primary h-4 w-20 rounded'} flex justify-end
```
Comment (schystz) [inline src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx]: Can we fix the indentation here and below line?

File: src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx [hunk 2]
```diff
@@ -102,26 +102,40 @@ const BridgeTransactionModal = ({
             <div className="flex justify-between mb-2 font-bold">
               <div className="text-secondary-alternate">Estimated lower bound shuttle fee</div>
               <div>
-                <div>
-                  {lowerBoundFee.toFixed(2)} {token.symbol}
+                <div
+                  className={`
+            ${!lowerBoundFee && 'animate-pulse bg-primary h-4 w-20 rounded'} flex justify-end
```
Comment (schystz) [inline src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx]: Can we fix the indentation here and below line?

File: src/pages/Tailwind/Swap/SwapDetails.tsx [hunk 3]
```diff
@@ -24,8 +24,12 @@ SwapDetailsProps) {
     <>
       <div className="flex flex-row justify-start mt-6 px-8 w-container text-sm font-bold">
         <div className="w-1/2 text-secondary-alternate">Price</div>
-        <div className="w-1/2 flex justify-end">
-          {price ? `${formatNumber(price, NumberFormat.long)} ${toCurrency}/${fromCurrency}` : '--'}
+        <div
+          className={`
+            ${!price && 'animate-pulse bg-primary h-4 w-36 rounded'} w-1/2 flex justify-end
```
Comment (schystz) [inline src/pages/Tailwind/Swap/SwapDetails.tsx]: Shall we add the loader too for Minimum Received? (line 37)

File: src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx (hunk 2 above)
Comment (schystz) [reply src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx]: Also can we add the loader effect on Gas Fee (line 99) as well, if applicable?

**Review by schystz — COMMENTED** [review]
Looks good in general. Just a few minor comments

File: src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx (hunk 1 above)
Comment (bill-ov) [reply src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx]: Done fixing the indentation of line 117-119 of BridgeTransactionModal.tsx

File: src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx (hunk 2 above)
Comment (bill-ov) [reply src/pages/Tailwind/Bridge/modals/BridgeTransactionModal.tsx]: Done fixing the indentation of line 105-107 of BridgeTransactionModal.tsx and added loader effect on Gas Fee

File: src/pages/Tailwind/Swap/SwapDetails.tsx (hunk 3 above)
Comment (bill-ov) [reply src/pages/Tailwind/Swap/SwapDetails.tsx]: Done adding loader for Minimum Received in SwapDetails.tsx

**Review by tracyarciaga — APPROVED** [review]

**Review by schystz — APPROVED** [review]
Looks good now. Thanks @psalm-ov !

**Pull request merged**